ci-operator/step-registry/rhcs/e2e/general-tests/rhcs-e2e-general-tests-ref.yaml

```diff
@@ -7,7 +7,7 @@ ref:
     requests:
       cpu: 100m
       memory: 300Mi
-  timeout: 3h0m0s
+  timeout: 6h0m0s
```

Comment:
The [timeout](https://github.com/openshift/release/blob/master/ci-operator/step-registry/rhcs/e2e/general-tests/rhcs-e2e-general-tests-commands.sh#L77) in the `ginkgo` command is set to `2h`. Since you are extending the running time, I think you may also need to extend it.